client/blocks/login/login-header.tsx

```diff
@@ -137,7 +135,7 @@ export function getHeaderText(
 					</>
 				);
 			}
-			headerText = translate( 'Log in to your account' );
+			headerText = translate( 'Log in to Woo with WordPress.com' );
```

Comment:
This should actually be handled already up on top inside the `isSocialFirst`: https://github.com/Automattic/wp-calypso/pull/104104/files#diff-fc758b906c257441858a1ebb10f7aa7360fb06563cd3962d011770a777cd2a61R75

Isn't that one picked up already? So no need to update here - other than remove completely? 🤔

Reply:
Sorry, forgot to reply to this. This is done now.